### DISCUSSION TITLE
How many sales/downloads has ToonSquid had?

### DISCUSSION THREAD

**Discussion by DrNomz** [keiwando/toonsquid#670] [Q&A] [answered]
Literally curious because I just learned this exists. It looks dope! I don't think I'll ever personally try it (no interest in Ipads or whatever) but @coggsie uses it and his animations are amazing. :)

**Comment by Lioncub901** [comment]
I tried asking something similar (can ToonSquid be a full time job). I don't think the dev wants to answer though.

**Reply by DrNomz** [reply]
Aw man really? Bruh I don't see why, interact with the community if you want to be cool. Come onnnnnnnnnnnnnnnnn broooooooooooooo not responding at all is BEYOND lame. Be a cool studio like the ones who made Aesprite. Those guys are COOL.

**Reply by Lioncub901** [reply, marked as answer]
Well that is not the dev’s style. He is a person that stay quiet but releases the best updates known to mankind. A man of secret.

**Reply by DrNomz** [reply]
Ha epic. Like Bing Soy lol. Aight fine, that's cool, I'll leave it be. :)

**Comment by Cat-Butler** [comment]
See [Troll (slang)](https://en.wikipedia.org/wiki/Troll_(slang)) ?

**Reply by DrNomz** [reply]
What!? xD lmao I don't get why but okay. Tempted to mark that as the answer. xD
